gradle.properties

```diff
@@ -1,28 +1,35 @@
-# IntelliJ Platform Artifacts Repositories
-# -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
-pluginGroup=intellij
-pluginName=intellij-buf
-pluginRepositoryUrl=https://github.com/bufbuild/intellij-buf
+# IntelliJ Platform Artifacts Repositories -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
```

Comment:
Added whitespace changes to make it easier to see the diffs between our version of this file and the upstream template. The only changed values are `pluginVersion` (to prepare for the next release) and `gradleVersion`.